spec/features/user_can_delete_post_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.feature "Delete post", type: :feature do
+  scenario "user can delete own post" do
+    log_in
+    post_picture
+    expect(first('.each-post')).to have_link "Delete"
+    click_link "Delete"
```

Comment:
This seems unusual to have 2 expect statements with another action in between the two, that would indicate that you are testing two separate scenarios within one test? Although I can't find anything concrete online to support that so I could be wrong

Reply:
I thought that this was okay with a feature test?

Reply:
I would say ok for a feature test. But you'll definitely see people with different opinions here.